strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/restart/RestartController.java

```diff
@@ -0,0 +1,34 @@
+package org.carlspring.strongbox.controllers.restart;
+
+import io.swagger.annotations.Api;
+import org.carlspring.strongbox.app.StrongboxSpringBootApplication;
+import org.carlspring.strongbox.controllers.BaseController;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.carlspring.strongbox.controllers.restart.RestartController.MAPPING;
+
+
+/**
+ * @author: adavid9
+ */
+@RestController
+@RequestMapping(value = MAPPING)
+public class RestartController
+        extends BaseController
+{
+
+    public static final String MAPPING = "/api/monitoring";
```

Comment:
@carlspring Maybe this should be under `/api/restart` ? (or if you have better ideas)

Reply:
+1: I agree!